menyer/pizza_house/assets/pizza_house.css

```diff
@@ -0,0 +1,127 @@
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    width: 100%;
+    max-height: 100%;
+    margin: 0;
+    padding: 0;
+}
+
+#main {
+    max-height: 100%;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    padding: 1em;
+}
+
+.information {
+    box-shadow: 0 2px 1px -1px rgba(0,0,0,.2), 0 1px 1px 0 rgba(0,0,0,.14), 0 1px 3px 0 rgba(0,0,0,.12);
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 3em;
+}
+
+.information h2,
+.information h1 {
+    margin: 0;
+    color: #cd212a;
+}
+
+.information h1 {
+    font-size: 48px;
+}
+
+.information img {
+    height: 128px;
+    width: 464px;
+}
+
+.cards {
+    width: 100%;
+    max-height: 100%;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-gap: 1em;
+}
+
+.container {
+    width: 100%;
+    max-height: 100%;
+    height: auto;
+    box-shadow: 0 2px 1px -1px rgba(0,0,0,.2), 0 1px 1px 0 rgba(0,0,0,.14), 0 1px 3px 0 rgba(0,0,0,.12);
+    display: flex;
+    flex-direction: column;
+    border-radius: 10px;
+    background: #f4f5f0;
+}
+
+
+.container header {
+    padding: 1em;
+    background-color: #008c45;
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    justify-content: space-between;
+}
+
+.container header h2 {
+    font-size: 44px;
+    margin: 0;
+    color: #ffffff;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+}
+
+
+
```

Comment:
Keep the number of new lines between selectors consistent. Recommended is just one.